examples/rust/BUILD

```diff
@@ -0,0 +1,13 @@
+# Public notice: this file is for internal documentation, testing, and
+# reference only. Note that repo maintainers can freely change any part of the
+# repository code at any time.
+package(default_visibility = ["//visibility:public"])
+
+# Rust
+load("@io_bazel_rules_docker//rust:image.bzl", "rust_image")
+
+rust_image(
+    name = "rust_example",
+    srcs = ["src/main.rs"],
+    base = "//cc:cc",
+)
```

Comment:
So this compiles for the current platform, which may be different from the container image's platform (e.g., compiled for macOS but deploying to Linux) which results in an odd error when launching the image:
```console
$ bazel run //examples/rust:rust_example
[...]
INFO: Build completed successfully, 1 total action
Loaded image ID: sha256:486acab411b4a2362f606e5167028cad10fcdcb02a549e18bc9068168b7061cd
Tagging 486acab411b4a2362f606e5167028cad10fcdcb02a549e18bc9068168b7061cd as bazel/examples/rust:rust_example
standard_init_linux.go:211: exec user process caused "exec format error"
```

Unfortunately the rust rules don't seem to support cross-compilation yet (bazelbuild/rules_rust#276, and bazelbuild/rules_docker#372), but there isn't a clean way to cause the build to fail (bazelbuild/bazel#3780).  So could you just add a note here that we require building on the same platform as the image.

Reply:
Argh that's annoying, I've added the note there as requested. Thanks for looking into that and explaining! I can imagine with WASM that will definitely be a pain point.